docs/common/ca.md

```diff
@@ -6,109 +6,87 @@ The [certificate authorities](https://en.wikipedia.org/wiki/Certificate_authorit
+        | :-------------------------------------- | :----------------------------------------   | :--------------------------------------------   |
+        | Start a service                         | `service <SERVICE-NAME> start` | `systemctl start <SERVICE-NAME>`   |
+        | Stop a  service                         | `service <SERVICE-NAME> stop`  | `systemctl stop <SERVICE-NAME>`    |
+        | Enable a service to start on boot       | `chkconfig <SERVICE-NAME> on`  | `systemctl enable <SERVICE-NAME>`  |
+        | Disable a service from starting on boot | `chkconfig <SERVICE-NAME> off` | `systemctl disable <SERVICE-NAME>` |
 
-Will update CA certificates from version unknown to version 1.21NEW.
-Update successful.
+1. (Optional) To add a new CA:
 
-Setup completed successfully.
-```
+        osg-ca-manage add [--dir <local_dir>] --hash <CA-HASH>
 
-Initially the CA certificates will not be updated. You can tell by looking at:
+1. (Optional) To remove a CA
 
-``` console
-root@host # /sbin/service osg-update-certs-cron  status
-Periodic osg-update-certs is disabled.
-```
+        osg-ca-manage remove --hash <CA-HASH>
 
-You can enable the `cron` job that updates the CA certs with:
-
-``` console
-root@host # /sbin/service osg-update-certs-cron  start
-Enabling periodic osg-update-certs:                        [  %GREEN%OK%ENDCOLOR%  ]
-```
-
-A complete set of options available though `osg-ca-manage` command, including your interface to adding and removing CAs, could be found at [osg-ca-manage documentation](/security/certificate-management)
+A complete set of options available though `osg-ca-manage` command, can be found in the [osg-ca-manage documentation](/security/certificate-management#managing-cas)
 
 ### Option 3: Site-managed CAs ###
 
-If you want to handle the list of CAs completely internally to the site, you can utilize the `empty-ca-certs` RPM to satisfy
-RPM dependencies - but not actually install any CAs.
-
-Install this with:
+If you want to handle the list of CAs completely internally to your site, you can utilize the `empty-ca-certs` RPM to satisfy
+RPM dependencies - but not actually install any CAs. To install this RPM, run the following command:
 
 ``` console
-yum install empty-ca-certs –-enablerepo=osg-empty
+root@host # yum install empty-ca-certs –-enablerepo=osg-empty
 ```
 
 !!! warning
-    If you choose this option, you are responsible for installing the CA certificates yourself. You must install them in `/etc/grid-security/certificates`, or make a symlink from that location to the directory that contains the CA certificates.
+    If you choose this option, you are responsible for installing and maintaing the CA certificates. They must be installed in `/etc/grid-security/certificates`, or make a symlink from that location to the directory that contains the CA certificates.
```

Comment:
Tense does not match -- either leave it as "you must install them" or change "make a symlink" to "a symlink must be made".